Rakefile

```diff
@@ -22,9 +21,73 @@ namespace :test do
   end
 end
 
-desc 'Build and publish the gem'
-task publish: :build do
-  system("gem push pkg/inspec_tools-#{InspecTools::VERSION}.gem")
+desc 'Build for release'
+task :build_release do
+
+  Rake::Task["generate_mapping_objects"].reenable
+  Rake::Task["generate_mapping_objects"].invoke
+
+  system('gem build *.gemspec')
```

Comment:
This should probably only build the inspec_tools.gemspec, not just any gemspec in the folder.

`system('gem build inspec_tools.gemspec')`